Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityCameraSettings.cs

```diff
@@ -39,10 +39,30 @@ public class WindowsMixedRealityCameraSettings : BaseCameraSettingsProvider
             BaseCameraSettingsProfile profile = null) : base(cameraSystem, name, priority, profile)
         { }
 
+        /// <inheritdoc/>
+        public override void Enable()
+        {
+            base.Enable();
+            InitializeReprojectionUpdater();
```

Comment:
Should we only add this if reprojection is set to autoplanar? Feels weird to potentially add a MonoBehaviour that won't do anything

Reply:
Though I suppose if we support changing the value at runtime, this is the only way?

Reply:
yep. we need to be able to respond to PostRender